src/components/Ctas.tsx

```diff
@@ -0,0 +1,56 @@
+import Button, { type ButtonProps } from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { isValidElement } from 'react';
+import { type CommonProps } from 'types/components';
+
+interface CtaButtonProps extends Partial<ButtonProps> {
+  text: string;
+}
+
+type CtaButton = CtaButtonProps | JSX.Element;
+
```

Comment:
Please, let's use the `Generic` type exported from 'src/types/components' instead `JSX.Element`